test/dlc_tests/conftest.py

```diff
@@ -128,6 +128,18 @@
     "feature_s3_plugin_present": {NightlyFeatureLabel.AWS_S3_PLUGIN_INSTALLED.value},
 }
 
+@pytest.fixture(autouse=True)
+def skip_s3plugin_test(request, pytorch_training):
+    if request.node.get_closest_marker("skip_s3plugin_test"):
+        if Version(pytorch_training) < Version("1.6.0") or Version(pytorch_training) > Version(
```

Comment:
We should probably use a `SpecifierSet` here - see explanation and example here https://github.com/aws/deep-learning-containers/blob/662a65243fec29b150749f1b98d2df1e68ae723c/test/dlc_tests/sanity/test_safety_check.py#L1093.